src/Robo/Commands/Validate/TwigCommand.php

```diff
@@ -26,35 +22,11 @@ public function lintFileSets() {
     $fileset_ids = $this->getConfigValue('validate.twig.filesets');
     $filesets = $fileset_manager->getFilesets($fileset_ids);
     $bin = $this->getConfigValue('composer.bin');
-    $command = "'$bin/twig-lint' lint --only-print-errors '%s'";
-    $this->executeCommandAgainstFilesets($filesets, $command, TRUE);
-  }
-
-  /**
-   * Executes Twig validator against a list of files, if in twig.filesets.
-   *
-   * @command validate:twig:files
-   *
-   * @param string $file_list
-   *   A list of files to scan, separated by \n.
-   */
-  public function lintFileList($file_list) {
-    $this->say("Linting twig files...");
-
-    $files = explode("\n", $file_list);
-
-    /** @var \Acquia\Blt\Robo\Filesets\FilesetManager $fileset_manager */
-    $fileset_manager = $this->getContainer()->get('filesetManager');
-    $fileset_ids = $this->getConfigValue('validate.twig.filesets');
-    $filesets = $fileset_manager->getFilesets($fileset_ids);
-
-    $bin = $this->getConfigValue('composer.bin');
-    $command = "'$bin/twig-lint' lint --only-print-errors '%s'";
-    foreach ($filesets as $fileset_id => $fileset) {
-      $filesets[$fileset_id] = $fileset_manager->filterFilesByFileset($files, $fileset);
-    }
-
+    $command = "'$bin/blt' 'validate:twig:files' '%s'";
```

Comment:
Aren't you removing the `validate:twig:files` command above? Doesn't seem like it makes sense to call it here.

Reply:
`validate:twig:files` gets added in `src/Robo/Blt.php` line 157.